mobile/library/cc/log_level.cc

```diff
@@ -18,7 +20,8 @@ std::string logLevelToString(LogLevel method) {
     }
   }
 
-  throw std::out_of_range("unknown log level type");
+  IS_ENVOY_BUG("unknown log level, defaulting to off");
+  return LOG_LEVEL_LOOKUP[6].second;
```

Comment:
Instead of the magic number 6, can we use ABSL_ARRAYSIZE(LOG_LEVEL_LOOKUP)?

I almost wrote $#LOG_LEVEL_LOOKUP before I realized what language we were writing in :P

Reply:
minus one but sure =P

Reply:
Hah! Good point :)